### PR TITLE
ticdc: add more info for config `output-old-value` in csv protocal

### PR DESCRIPTION
**Pull request by hongyunyan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add more doc info for the config output-old-value, which is introduced in v7.6

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tiflow/pull/10542, https://github.com/pingcap/docs-cn/pull/16874

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by fubinzh — APPROVED** [review]
/lgtm

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -208,6 +208,8 @@ enable-partition-separator = true
 # 是否输出 handle 列信息。默认值为 false。该配置项仅用于内部实现，不推荐设置该配置项。
 # output-handle-key = false
 # 是否输出行数据更改前的值。默认值为 false。开启后，Update 事件会输出两行数据：第一行为 Delete 事件，输出更改前的数据；第二行为 Insert 事件，输出更改后的数据。
+# 当该参数设为 true 时，会在变更数据列前增加 "is-update" 列。该列用来标识当前行的变更数据时来自 Update 事件，还是原始的 Insert/Delete 事件。
```
Comment (lilin90) [inline ticdc/ticdc-changefeed-config.md:211]:
```suggestion
# 当该参数设为 true 时，会在变更数据列前增加 "is-update" 列。该列用来标识当前行的变更数据是来自 Update 事件，还是原始的 Insert/Delete 事件。
```

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -208,6 +208,8 @@ enable-partition-separator = true
...
+# 当当前行的变更数据来自 Update 事件，则该列为 true，否则则为 false。
```
Comment (lilin90) [inline ticdc/ticdc-changefeed-config.md:212]:
```suggestion
# 如果当前行的变更数据来自 Update 事件，则 "is-update" 列为 true，否则为 false。
```

File: ticdc/ticdc-csv.md [hunk 3]
```diff
@@ -48,7 +49,8 @@ CSV 文件中，单行的每一列定义如下：
 - 第二列：表名。
 - 第三列：库名。
 - 第四列：`commit ts`，即原始事务的 commit ts。该列为可选配置。
-- 第五列至最后一列：变更数据的列，可为一列或多列。
+- 第五列: `is-update`，该列仅在 `output-old-value` 为 true 时存在，用于标识该行变更来自 Update 事件（值为 true 时），还是来自 Insert/Delete 事件（值为 false）。
```
Comment (lilin90) [inline ticdc/ticdc-csv.md:52]:
```suggestion
- 第五列：`is-update`，该列仅在 `output-old-value` 为 true 时存在，用于标识该行变更来自 Update 事件（值为 true），还是来自 Insert/Delete 事件（值为 false）。
```

File: ticdc/ticdc-csv.md [hunk 4]
```diff
@@ -62,7 +64,7 @@ CREATE TABLE `employee` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 ```
 
-该表上的 DML 事件以 CSV 格式存储后如下所示：
+当配置中 `include-commit-ts = true`，`output-old-value = false`时，该表上的 DML 事件以 CSV 格式存储后如下所示：
```
Comment (lilin90) [inline ticdc/ticdc-csv.md:67]:
```suggestion
当配置中 `include-commit-ts = true` 且 `output-old-value = false` 时，该表上的 DML 事件以 CSV 格式存储后如下所示：
```

File: ticdc/ticdc-csv.md [hunk 5]
```diff
@@ -72,6 +74,18 @@ CREATE TABLE `employee` (
 "U","employee","hr",433305438660591630,102,"Alex","Alice","2018-06-15","Beijing"
 ```
 
+当配置中 `include-commit-ts = true`，`output-old-value = true`时，该表上的 DML 事件以 CSV 格式存储后如下所示：
```
Comment (lilin90) [inline ticdc/ticdc-csv.md:77]:
```suggestion
当配置中 `include-commit-ts = true` 且 `output-old-value = true` 时，该表上的 DML 事件以 CSV 格式存储后如下所示：
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by hfxsd — APPROVED** [review]

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**